README.md

```diff
@@ -1,7 +1,42 @@
 
-# country-by-country-reporting
+# Country by Country Reporting
 
-This is a placeholder README.md for a new repository
+Backend service to support file uploads from registered users (agent, client or organisation) on behalf of an organisation.
+
+This service integrates with HOD, i.e. DES/ETMP.
+
+The frontend to this service can be found [here]("https://github.com/hmrc/country-by-country-reporting-frontend/")
```

Comment:
Rich diff not showing this a link perhaps the quotes should be removed as in DAC6-2257